src/forms/ErrorList.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+export const ERROR_CLASSNAME = 'text--error text--small';
+
+const ErrorList = (props) => {
+	const {
+		errorId,
+		errors,
+		...other
+	} = props;
+
+	const errorList = Array.isArray(errors) ? errors : [errors];
+
+	if (errorId) {
+		other['id'] = errorId;
```

Comment:
ESLint might not be set up on this, but we're using a rule that will error when using square brackets unnecessarily in mup-web - this is equivalent to `other.id = errorId`